app/components/agenda/agenda-header/component.js

```diff
@@ -140,14 +143,16 @@ export default Component.extend(isAuthenticatedMixin, FileSaverMixin, {
         .filter((agenda) => agenda.name !== 'Ontwerpagenda')
         .sortBy('-name')
         .get('firstObject');
+        
+      const session = await lastAgenda.get('createdFor');
+      session.set('isFinal', true);
+      session.set('agenda', lastAgenda);
+      await session.save();
```

Comment:
Ik zou dit pas doen wanneer je zeker weet dat de agenda verwijderd is.
Indien er dan iets misloopt bij het verwijderen van de ontwerpagenda gaat de zitting niet afgesloten zijn, waardoor je ze weer opnieuw kan proberen af te sluiten.